tools/Statistics/CGAL/CGALTypes.h

```diff
@@ -0,0 +1,26 @@
+#ifndef PYMESH_CGALTYPES_H
+#define PYMESH_CGALTYPES_H
+
+#include <CGAL/Exact_predicates_exact_constructions_kernel.h>
+#include <CGAL/array.h>
+#include <CGAL/Polyhedron_3.h>
+#include <CGAL/Surface_mesh.h>
+#include <CGAL/Simple_cartesian.h>
+#include <CGAL/Nef_polyhedron_2.h>
+#include <CGAL/Point_3.h>
+
+typedef std::array<double, 3>                           Point_type;
+typedef std::array<int, 3>                              Polygon_type;
+typedef CGAL::Simple_cartesian<double>                  Kernel;
+typedef CGAL::Polyhedron_3<Kernel>                      Polyhedron;
```

Comment:
What are you using `Polyhedron_3` for? More precisely, why `Surface_mesh` isn't enough?

Reply:
I have been was adapting some code to use (see 879c4e99abd29bd18b02c2c7add12ae04efd49de), and should be able to remove `Polyhedron_3` entirely.

I need to get in this PR again.